### ISSUE TITLE
get_config() fails with UnboundLocalError for several arguments

### ISSUE THREAD

**Issue by brunocous** [pycaret/pycaret#678]
Using Pycaret 2.1.2 in a notebook and in a conda env for a regression task, `get_config()` fails for several arguments with an `UnboundLocalError` for a few arguments (Y_test, Y_train, prep_pipeline) defined in [https://pycaret.org/regression/#get-config](https://pycaret.org/regression/#get-config). Weirdly, it works for X_train and X_test for example. 
````
INFO - Initializing get_config()
INFO - get_config(variable=prep_pipeline)
INFO - Global variable: prep_pipeline returned
INFO - get_config() succesfully completed......................................
---------------------------------------------------------------------------
UnboundLocalError                         Traceback (most recent call last)
<ipython-input-64-ff5ebe265848> in <module>
----> 1 get_config('prep_pipeline')

/anaconda/envs/azureml_py36/lib/python3.6/site-packages/pycaret/regression.py in get_config(variable)
   9138     logger.info("get_config() succesfully completed......................................")
   9139 
-> 9140     return global_var
   9141 
   9142 def set_config(variable,value):

UnboundLocalError: local variable 'global_var' referenced before assignment
````

This is a scoping issue, but don't know how to quick fix it in my local notebook.

**Comment by Yard1** [comment]
Will be fixed in the next release.

**Comment by pycaret** [comment]
@brunocous 

This issue has been fixed in the `pycaret-nightly` release. Please install pycaret-nightly using `pip install pycaret-nightly`. Feel free to open a new issue, in case the problem persists. This fix will be published to PyPi as `pycaret==2.2.0` on October 28, 2020.
